src/isar/models/communication/status.py

```diff
@@ -10,7 +10,6 @@
 class Status:
     mission_status: Optional[MissionStatus]
     mission_in_progress: bool
-    current_mission_instance_id: Optional[int]
     current_mission_task: Optional[Task]
```

Comment:
Can we rename this to simply `current_task` while we're at it? The concept of `current_task` should be unique in this context now.